packages/perseus-editor/src/widgets/input-number-editor.jsx

```diff
@@ -45,22 +46,44 @@ const answerTypes = {
     },
 };
 
-type Props = $FlowFixMe;
+type Props = {|
+    value: number,
+    simplify: React.ElementConfig<typeof InputNumber.widget>["simplify"],
```

Comment:
I opted to use `React.ElementConfig` instead of `PerseusInputNumberWidgetOptions` because `PerseusInputNumberWidgetOptions` isn't exported by Perseus. I think this is valid though as this editor wants to manage editing the props that are passed into the `InputNumber` component.

Reply:
`ElementConfig` is almost always a better option.